JS-Algorithms/challenges.js

```diff
@@ -70,7 +109,10 @@ The last 3 digits for the sum of powers from 1 to 10 is "317"
 ***** */
 
 const ownPower = (number, lastDigits) => {
-  // YOUR CODE HERE...
+  const iterationNumber = Array(number).fill().map((_, index) => index + 1)
+  const ownPowersValues = iterationNumber.reduce((prevValue, currentValue) => BigInt(prevValue) + (BigInt(currentValue) ** BigInt(currentValue)));
```

Comment:
It seems we have the initial value for the `reduce` missing, could you fix it please?